scripts/vm/network/vnet/modifyvxlan-evpn.sh

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#
+# Use BGP+EVPN for VXLAN with CloudStack instead of Multicast
+#
+# The default 'modifyvxlan.sh' script from CloudStack uses Multicast instead of EVPN for VXLAN
+# In order to use this script and thus utilize BGP+EVPN, symlink this file:
+#
+# ln -s /usr/share/modifyvxlan.sh /usr/share/cloudstack-common/scripts/vm/network/vnet/modifyvxlan-evpn.sh
+#
+#
+# CloudStack will not handle the BGP configuration nor communication, the operator of the hypervisor will
+# need to configure the properly.
+#
+# Frrouting is recommend to be used on the hypervisor to establish BGP sessions with upstream routers and 
+# exchange BGP+EVPN information.
+#
+# More information about BGP and EVPN with FRR: https://vincent.bernat.ch/en/blog/2017-vxlan-bgp-evpn
+#
```

Comment:
maybe this can be added to the usage output?

Reply:
I have added it to the documentation PR I have just opened. I don't think the "usage" is the proper location for sharing such a link. This link can also die in the future.

https://github.com/apache/cloudstack-documentation/pull/444